apps/web/src/server/applications/case/documentation/applications-documentation.controller.js

```diff
@@ -621,12 +628,12 @@ export async function updateFolderCreate(request, response) {
 			request.query,
 			request.session
 		);
-
 		return response.render(`applications/components/folder/folder`, {
 			...properties,
-			errors: errors || { msg: 'Something went wrong. Please, try again later.' }
+			errors: [errors] || [{ msg: 'Something went wrong. Please, try again later.' }]
 		});
 	}
 
-	response.redirect('./');
+	setSessionBanner(session, '<strong>Folder created</strong>');
```

Comment:
can the bold be done with CSS or by passing a config to `govukNotificationBanner`?